internal/pkg/aws/ecs/ecs.go

```diff
@@ -51,3 +183,23 @@ func (t *TaskDefinition) EnvironmentVariables() map[string]string {
 	}
 	return envs
 }
+
+// shortTaskID parses the task ARN and returns the short task ID.
+// For example: arn:aws:ecs:us-west-2:123456789:task/my-project-test-Cluster-9F7Y0RLP60R7/4082490ee6c245e09d2145010aa1ba8d
+// becomes 4082490.
+func shortTaskID(taskArn string) (string, error) {
```

Comment:
Might as well throw these on as private methods to the Task struct, since we have it!